README.md

```diff
@@ -1,112 +1,108 @@
+    Creating nebula-docker-compose_graphd_1    ... done
+    Creating nebula-docker-compose_storaged0_1 ... done
+    Creating nebula-docker-compose_storaged1_1 ... done
+    ```
 
-```shell
-$ docker-compose pull
-```
+    >**NOTE**: For more information of above services, see [Design and Architecture of Nebula Graph](https://docs.nebula-graph.io/manual-EN/1.overview/3.design-and-architecture/1.design-and-architecture/).
 
-5. Connect to the graph service of **Nebula Graph**.
+4. Use nebula-console to connect to **Nebula Graph**.
 
-```shell
-$ docker run --rm -ti --network=host vesoft/nebula-console:nightly --addr=127.0.0.1 --port=3699
-```
+    Nebula-console is the native CLI client of **Nebula Graph**. Docker pulls the nebula-console images automatically from Docker Hub according to the path we set in the following commands, and uses it to connect to the graphd service of **Nebula Graph**.
 
-If the authorization is not enabled, **Nebula Graph** will use the default user name and password to log in, so you can skip the user and password specification here. If the authorization is enabled, the user and password must be specified. For example:
+   * For **Nebula Graph** 1.0:
 
-Set the `enable_authorize` parameter in the `/usr/local/nebula/etc/nebula-graphd.conf` file to `true` to enable the authorization. In this case, you must provide the user and password to connect.
+    ```shell
+    $ docker run --rm -ti --network=host vesoft/nebula-console:nightly -u <user> -p <password> --addr=127.0.0.1 --port=3699
+    ```
 
-```shell
-$ docker run --rm -ti --network=host vesoft/nebula-console:nightly -u <user> -p <password> --addr=127.0.0.1 --port=3699
-```
+   * For **Nebula Graph** 2.0 pre:
 
-The following information which indicates that you successfully connect to **Nebula Graph** is displayed:
+    ```shell
+    $ docker run --rm -ti --network nebula-docker-compose_nebula-net vesoft/nebula-console:v2-preview-nightly -u <user> -p <password> --address=graphd --port=3699
+    ```
 
-```shell
-Welcome to Nebula Graph (Version 5d10861)
+    >**NOTE**: By default, the authentication is disabled, and the `-u` and `-p` options are unnecessary. To enbale authentication, see [Authentication Configurations](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/4.account-management-statements/authentication/#authentication).
 
-(user@127.0.0.1) [(none)]>
+    The following information is displayed when you have connected to the **Nebula Graph** services:
 
-```
+    ```shell
+    Welcome to Nebula Graph (Version 5d10861)
 
-**Note**: Now, you can start using **Nebula Graph** by creating spaces, tags and more. For details, refer to [get started](https://docs.nebula-graph.io/manual-EN/1.overview/2.quick-start/1.get-started/).
+    (user@127.0.0.1) [(none)]>
+    ```
 
-## Checking the Status and Ports of **Nebula Graph** Services
+## Check the **Nebula Graph** Service Status and Ports
```

Comment:
when the nebula graph services are started, users can do this check, so I think a link to this part should be added to the "deploy" part. 

this check could help new users know why the 3699 port is used for nebula console connection.